package.json

```diff
@@ -3,7 +3,7 @@
   "license": "LGPL v3",
   "scripts": {
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
-    "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
+    "watch": "watch -p 'static/sass/*.scss' -c 'yarn run build'",
```

Comment:
I'm curious as to why you removed the recursive watcher?

Reply:
This only watches files in the root scss directory. The main styles should be contained in a patterns  directory but in this case it's a custom directory.

Reply:
I'm a tiny bit concerned that someone may add a sub dir to that sass dir and it won't properly build and won't know why.

If there are no sub directories then keeping the `**/*` doesn't have any negative effects, does it?

Reply:
Fair point, I've reverted for now.